macros/edr/metadata_collection/get_columns_by_configured_schemas.sql

```diff
@@ -0,0 +1,7 @@
+{% macro get_columns_by_configured_schemas(configured_schemas) %}
```

Comment:
Maybe it should be just `get_columns_by_schemas`?
(I think this function is not aware that they're "configured")